docs/operating-eck/air-gapped.asciidoc

```diff
@@ -33,6 +33,23 @@ To make use of your mirrored images you can either set the image for each applic
 
 
 
+[float]
+[id="{p}-use-internal-operator-image"]
+== Use an internal image of the ECK operator
+
+To deploy the ECK operator in an air-gapped environment, you first have to mirror the operator image itself from `docker.elastic.co` to a private container registry, for example `my-registry`.
+
+Once the ECK operator image has been copied, you have to replace the original image name `docker.elastic.co/eck/eck-operator:{eck_version}` with the private name of the image, for example `my-registry/eck/eck-operator:{eck_version}`, in the <<{p}-install-yaml-manifests,operator manifests>>. When using <<{p}-install-helm,Helm charts>>, you have to replace the `image.repository` Helm value with, for example, `my-registry/eck/eck-operator`.
+
+Container registry vendors generally offer native solutions to mirror container images internally. As an alternative, you can use a Docker client which has access to both the public `docker.elastic.co` registry and to the private container registry as follows:
+
+[source,sh]
+----
+docker pull docker.elastic.co/eck/eck-operator:{eck_version}
+docker tag docker.elastic.co/eck/eck-operator:{eck_version} my-registry/eck/eck-operator:{eck_version}
+docker push my-registry/eck/eck-operator:{eck_version}
+----
+
```

Comment:
Last time I remember we went to the consensus that it's better to hide the concrete command for docker because normally people should know how to use container runtime to do things, i.e. pull/save/load/push container images.
I don't mind adding this to the doc because it explains things clearer, but just want to double check with you if it's really that you would like to add, or an accident...?

Reply:
As I found the PR to be a bit "thin", I added this section for additional informations. However you are right we went to a consensus. Let's remove it